sdk/core/src/lib.rs

```diff
@@ -3,6 +3,10 @@
 //! This crate is part of the unofficial Azure SDK effort in Rust. For more
 //! information on the project, and an overview of other crates, please refer to
 //! [our GitHub repository](https://github.com/azure/azure-sdk-for-rust).
+//!
+//! It is a library that provides cross-cutting services to other client
+//! libraries.  Please see the [general
+//! guidelines](https://azure.github.io/azure-sdk/general_azurecore.html).
```

Comment:
Why not just use `#![include_str!("../README.md")]`? Or is the assumption that the README may grow (for officialness, it mostly likely will have to) and we don't necessarily want this to grow the same?

Reply:
If we look at the README used in other SDKs, they include things we don't need as part of crate documentation but are entirely reasonable for the git repo, such release processes (how we define preview, ga, etc), contribution guidelines, security reporting guidelines, code of conduct for the repo, etc.